test/unit-complete/variables.tf

```diff
@@ -195,6 +195,18 @@ variable "secret_text" {
   default     = "42"
 }
 
+variable "encrypted_secret_name" {
+  description = "The name of the secret."
+  type        = string
+  default     = "ENCRYPTEDSECRET"
+}
+
+variable "encrypted_secret_text" {
+  description = "Secret value in Base64 format."
+  type        = string
+  default     = "NDI="
+}
```

Comment:
base64 of a plain value is not an encrypted value. it should be actually encrypted and base64 encoded.

Reply:
is there a way to do so with terraform only using the public key and encrypting values?

e.g. with 
https://registry.terraform.io/providers/integrations/github/latest/docs/data-sources/actions_public_key ?

Reply:
the provider do not accept a non-base64 value for `encrypted_value` so is it OK to leave the value as it is for testing purposes?